.github/workflows/daily-deploy-production.yml

```diff
@@ -20,6 +20,19 @@ env:
   BUILD_ENV: vagovprod
 
 jobs:
+  get-workflow-environment:
+    runs-on: ubuntu-latest
+    outputs:
+      environment_name: ${{ steps.check-environment.outputs.env_name }}
+    steps:
+      - name: Check environment
+        id: check-environment
+        run: |
+          if [[ ${{ github.event_name }} == 'workflow_dispatch' ]]; then
+            echo ::set-output name=env_name::'dev-test'
```

Comment:
Should we change the name of the environment to `production` now that we're not testing this anymore?

Also, we might not need the `else` statement that sets the output string as empty. If there's no output set for a variable, I think it defaults to an empty string. If that's the case, we can probably just use an `if` statement in GHA. Something like:
``` yaml
if: github.event_name == 'workflow_dispatch'
run: echo ::set-output name=env_name::'production'
```

Reply:
updated